Mathlib/Tactic/Linter/AttributeInstanceIn.lean

```diff
@@ -41,6 +41,34 @@ attribute [ext] who in
 
 example {x y : Nat} : x = y := by ext
 ```
+
+For *removing* instances, the `in` seems to work as expected.
```

Comment:
I think this wording is overly strong; `attribute [-foo] in` very explicitly uses a feature of scoped attributes to locally remove the attribute within the current section.

Reply:
```suggestion
For *removing* instances, the `in` works as expected.
```